source/generate_uids/generate_storage_sopclass_uids.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+"""
+Create UIDs for the Storage SOP Classes taken from the generated ``_uids.py``.
+"""
+import os
+import re
+import tokenize
+
+import sys
+
+import pydicom
+
+from pydicom._uid_dict import UID_dictionary
+
+STORAGE_REGEX = re.compile('.*(Storage|Storage SOP Class|Storage - '
+                           'For Presentation|Storage - For Processing)$')
+
+IS_PYTHON3 = sys.version_info >= (3,)
```

Comment:
lol, yeah, after I wrote that I thought maybe I should have hedged that bet for a longer time horizon.  In any case, in my mind we are already coding for python 3, with python 2 being the afterthought we make allowances for.